### DISCUSSION TITLE
Allow to add unstarred repos to a list

### DISCUSSION THREAD

**Discussion by 01231** [community/community#10272] [Other Feature Feedback, Questions, & Ideas]
**IS:** Adding repos to a list automatically stars them

**Wish:** Adding a repo to a list shouldn't change its state

**Comment by BenjaminMichaelis** [comment]
Totally agree, being able to organize repositories into lists on their own would be awesome, even if they aren't starred. Would help clean up my own repositories page greatly which would provide almost a folder like organization for repositories, would be very nice.
Would solve this issue better thats been ongoing for over 9 years:
[Can I arrange repositories into folders on Github?](https://stackoverflow.com/questions/11852982/can-i-arrange-repositories-into-folders-on-github#:~:text=Original%20answer%3A%20On%20GitHub%20itself%2C%20you%20cannot%20group,as%20a%20group%20of%20all%20sublimeText%20packages%20repos.)

**Reply by BenjaminMichaelis** [reply]
This is the current behavior:

https://user-images.githubusercontent.com/22186029/150289756-265737cc-745e-439c-a5b8-882aa4d8af14.mp4

**Comment by rkechols** [comment]
I would also like to add my voice of thorough agreement.
Lists are so close to the personal repo organization that we've all been waiting for... but the dumb stars.
To me, starring a repo is something completely different from putting it in a list or giving it a tag.

**Comment by volkovro** [comment]
**I'm joining you.** 
Automatic stared is a terrible solution. Instead of conveniently splitting repositories into lists, I give myself stars :(
_By the way, this does not happen with repositories without stars or with the first one added to the list - need to check it_

**Reply by kubo6472** [reply]
correct, same behavior that I noticed, glad it wasn't a fluke.
